src/it/JENKINS-45740-metadata/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>2.28</version>
+        <relativePath/>
+    </parent>
+    <groupId>org.jenkins-ci.tools.hpi.its</groupId>
+    <artifactId>JENKINS-45740-metadata</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>hpi</packaging>
+    <properties>
+        <hpi-plugin.version>@project.version@</hpi-plugin.version>
+        <plugin.info.changelogUrl>http://plugins.jenkins.io/JENKINS-45740-metadata/changelog</plugin.info.changelogUrl>
+        <plugin.info.logoUrl>http://plugins.jenkins.io/JENKINS-45740-metadata/logo.png</plugin.info.logoUrl>
+    </properties>
+    
+    <licenses>
+      <license>
+        <name>MIT License</name>
+        <url>http://www.opensource.org/licenses/mit-license.php</url>
+        <distribution>repo</distribution>
+      </license>
+      <license>
+        <name>MY License</name>
+        <url>http://mylicense.txt</url>
```

Comment:
How can we supporting two licenses? This seems like a nonexistent use case.

Reply:
AFAIK there is no dual-licensed plugins. I do not recall why I did that, but we could revert it to a single-licence mode to K.I.S.S.

Reply:
Yes I would suggest removing something we do not think we actually need, unless and until a clear need arises.